src/ApparentHorizons/StrahlkorperGr.hpp

```diff
@@ -207,6 +207,27 @@ double surface_integral_of_scalar(
     const Scalar<DataVector>& area_element, const Scalar<DataVector>& scalar,
     const Strahlkorper<Frame>& strahlkorper) noexcept;
 
+/*!
+ * \ingroup SurfacesGroup
+ * \brief Euclidean surface integral of a vector on a 2D `Strahlkorper`
+ *
+ * \details Computes the surface integral
+ * \f$\oint V^i s_i/\sqrt{s_i s_j \delta^{ij}} d^2S\f$ for a
```

Comment:
I find it easy to misread the documentation (I thought the `/` was a typo and not indicating division).  I would suggest either using `frac` or putting the denominator as `(...)^{-1/2}`

Reply:
sorry, just one more thing, I would prefer the formulas not have two sets of contracting `i` indices, please change one of them to another letter...